sds_data_manager/constructs/efs_construct.py

```diff
@@ -184,7 +184,7 @@ def __init__(
             allow_all_outbound=True,
             runtime=aws_lambda.Runtime.PYTHON_3_12,
             code=code,
-            handler="lambda_function.lambda_handler",
+            handler="efs_lambda.lambda_function.lambda_handler",
```

Comment:
Why are these changes in here?

Reply:
As I mentioned in Slack, this was another new bug I found. This line change fixes that issue.